src/contexts/accounts.tsx

```diff
@@ -203,8 +208,8 @@ const AccountsProvider: React.FC<PropsWithChildren> = ({ children }) => {
       };
     },
     {
-      enabled: Boolean(publicKey?.toString() && connected),
-      refetchInterval: 10_000,
+      enabled: Boolean(publicKey?.toString() && connected && getTerminalInView()),
+      refetchInterval: refetchIntervalForTokenAccounts,
```

Comment:
we should still keep 10s as default

Reply:
have set default value of `refetchIntervalForTokenAccounts` to 10s